SitecorePackageDeployer.Web/sitecore/admin/InstallAllPackages.aspx.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Hhogdev.SitecorePackageDeployer.Tasks;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Hhogdev.SitecorePackageDeployer.Entities;
+
+namespace Hhogdev.SitecorePackageDeployer.Web.sitecore.admin
+{
+    public partial class InstallAllPackages : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+            if (Request.QueryString["force"] == "1")
+            {
+                InstallPackage.ResetInstallState();
```

Comment:
Note to self- this should also clear that XML file that _can_ appear in Deploy folder.